runtime/include/chpl-tasks.h

```diff
@@ -104,6 +130,9 @@ void chpl_task_exit(void);        // called by the main task
 // thread) in order to be responsive and not be held up by other
 // user-level tasks. returns 0 on success, nonzero on failure.
 //
+// The caller of this function is responsible for ensuring that
+// *arg remains available to the task as long as it is running.
```

Comment:
You might say "... as long as it is needed" here instead.  In actuality, both comm=gasnet and comm=ugni send a NULL for the arg and don't use it at all.